### PR TITLE
Sequelize HowTo

### PR DESCRIPTION
**Pull request by mitchell-elholm**
By submitting this pull request, I confirm that my contribution is made under 
the terms of the MIT-0 license.

Thank you for your contribution!

### CODE REVIEW

File: .github/workflows/typescript-sequelize-integ-tests.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ secrets.TYPESCRIPT_IAM_ROLE }}
+        aws-region: us-east-1
+
+    - name: Configure and run integration for Sequelize
+      working-directory: ./typescript/sequelize
+      env:
+        CLUSTER_ENDPOINT: ${{ secrets.TYPESCRIPT_SEQ_CLUSTER_ENDPOINT }}
+        REGION: ${{ secrets.TYPESCRIPT_SEQ_CLUSTER_REGION }}
```
Comment (alinaliBQ) [inline .github/workflows/typescript-sequelize-integ-tests.yml:54]:
```suggestion
        CLUSTER_ENDPOINT: ${{ secrets.TYPESCRIPT_SEQUELIZE_CLUSTER_ENDPOINT}}
        REGION: ${{ secrets.TYPESCRIPT_SEQUELIZE_CLUSTER_REGION}}
```

File: .github/workflows/typescript-sequelize-integ-tests.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  # Give us a button to allow running the workflow on demand for testing.
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: 'Manual Workflow Run'
+        required: false
+        type: string
+
+jobs:
+  test:
```
Comment (alinaliBQ) [inline .github/workflows/typescript-sequelize-integ-tests.yml:25]:
```suggestion
  sequelize-integ-test:
```

**Review by alinaliBQ — APPROVED** [review]
LGTM

**Review by lmarcosi — APPROVED** [review]

**Review by alinaliBQ — CHANGES_REQUESTED** [review]
Please also update the root [README.md](https://github.com/aws-samples/aurora-dsql-samples/blob/main/README.md) to include the TypeScript HowTos

**Review by alinaliBQ — APPROVED** [review]

**Pull request merged**